spec/requests/api/v0/comments_spec.rb

```diff
@@ -167,7 +167,9 @@ def find_child_comment(response, action = :index)
     context "when getting by podcast episode id" do
       let(:podcast) { create(:podcast) }
       let(:podcast_episode) { create(:podcast_episode, podcast: podcast) }
-      let!(:comment) { create(:comment, commentable: podcast_episode) }
+      let(:comment) { create(:comment, commentable: podcast_episode) }
```

Comment:
Rubocop complained about `let!` when I tried to commit this so I fixed it.

Reply:
Ugh yeah, i kind of wish rubocop wouldn't because you end up having to invoke the dang thing just to make it not mad!